appveyor/test_task.bat

```diff
@@ -89,9 +89,20 @@ if not exist "%PHP_BUILD_CACHE_ENCHANT_DICT_DIR%\en_US.aff" (
 mkdir %LOCALAPPDATA%\enchant\hunspell
 copy %PHP_BUILD_CACHE_ENCHANT_DICT_DIR%\* %LOCALAPPDATA%\enchant\hunspell
 
-set TEST_PHPDBG_EXECUTABLE=%PHP_BUILD_OBJ_DIR%\Release
-if "%THREAD_SAFE%" equ "1" set TEST_PHPDBG_EXECUTABLE=%TEST_PHPDBG_EXECUTABLE%_TS
-set TEST_PHPDBG_EXECUTABLE=%TEST_PHPDBG_EXECUTABLE%\phpdbg.exe
+set PHP_BUILD_DIR=%PHP_BUILD_OBJ_DIR%\Release
+if "%THREAD_SAFE%" equ "1" set PHP_BUILD_DIR=%PHP_BUILD_DIR%_TS
+
+rem overwrite tmp-php.ini
+echo extension_dir=%PHP_BUILD_DIR% > %PHP_BUILD_DIR%\tmp-php.ini
+echo opcache.file_cache=%PHP_BUILD_DIR%\test_file_cache >> %PHP_BUILD_DIR%\tmp-php.ini
```

Comment:
Is that part of the old tmp-php.ini? So our opcache test on Windows is actually testing file cache and not SHM cache?

Reply:
`opcache.file_cache` does not disable SHM caching on Windows; that would require `opcache.file_cache_only` to be enabled (but defaults to off). The file cache is relevant for `opcache.file_cache_fallback` which is enabled by default.

Reply:
Right, that makes sense.